tests/integration/records/single_db/test_records_load.py

```diff
@@ -146,5 +146,5 @@ def load(self, format_type, variant, hints={}, broken=False, sourcefn=None):
             self.assertIn(out.move_count, [None, 1])
 
     def verify_db_table(self, variant):
-        validator = RecordsTableValidator(self.engine)
-        validator.validate(variant, schema_name=self.schema_name, table_name=self.table_name)
+        validator = RecordsTableValidator(self.engine, file_variant=variant)
```

Comment:
I'm clarifying with the terminology 'file_variant' that this is a source file (which may be loaded into pandas and turned into some other variant for loading - the `load_variant`).

Note that as Parquet support gets more fully integrated and tested we'll want to change these variant arguments into records_format arguments.